docs/src/managed/features.md

```diff
@@ -0,0 +1,34 @@
+# Features
+
+This page will cover configuration of various OKE features in CAPOCI.
```

Comment:
NIT: Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)
I thought it was Oracle Kubernetes Engine, but looks like the official docs have changed. https://www.oracle.com/cloud/cloud-native/container-engine-kubernetes/

Reply:
fixed